examples/rc-cpy-final.py

```diff
@@ -14,182 +15,163 @@
 from digitalio import DigitalInOut, Direction
 from pulseio import PWMOut, PulseIn, PulseOut
 
-## Customisation these variables
+import adafruit_logging as logging
```

Comment:
Does this need a separate library to be installed on the robo HAT?

Reply:
Yes, need to put a library on the lib folder